string srcAddress
string destAddress
uint16 port
bool isMulticast
---
bool successful
